pkg/controller/provider/model/ovirt/model.go

```diff
@@ -131,6 +131,7 @@ type VM struct {
 	CpuCores                    int16            `sql:""`
 	CpuThreads                  int16            `sql:""`
 	CpuAffinity                 []CpuPinning     `sql:""`
+	CpuPinningPolicy            string           `sql:""`
```

Comment:
that's a relatively recent addition to the api - does the "infrastructure" handle it well when it's missing?

Reply:
i'm pretty sure it is. for example you may have or not the `CpuAffinity` above. Also from my other PR, I don't remember exactly if it shows empty or nil.